keras/backend/cntk_backend.py

```diff
@@ -443,7 +455,8 @@ def count_params(x):
         if _ == C.InferredDimension or _ == C.FreeDimension:
             raise ValueError('CNTK backend: count_params with '
                              'InferredDimension and FreeDimension '
-                             'is not supported for now.')
+                             'is not supported. Please provide '
+                             'specified dimension.')
```

Comment:
This still message will still sound quite obscure for most users, as it refers to many unknown concepts, like `InferredDimension`, etc. Recommendation is still unclear to me